lib/xcodeproj/scheme/execution_action.rb

```diff
@@ -3,56 +3,60 @@ class XCScheme
     # This class wraps the ExecutionAction node of a .xcscheme XML file
     #
     class ExecutionAction < XMLElementWrapper
+      # @param [String] action_type
```

Comment:
why do we not prefer passing the action type as the symbol used in the constants hash?

Reply:
I missed that, makes sense to me 👍